deploy/operator.yaml

```diff
@@ -22,6 +22,8 @@ spec:
               cpu: "250m"
               memory: "512Mi"
           env:
+            - name: NO_PROXY
```

Comment:
Don't we need it for the core/db pods too?

Reply:
I've set it for core. I can do it for DB as well, but not sure we need